sympy/core/function.py

```diff
@@ -614,7 +614,7 @@ def _eval_derivative(self, s):
         return Add(*l)
 
     def _eval_is_commutative(self):
-        return fuzzy_and(a.is_commutative for a in self.args)
+        return True
```

Comment:
This method should be removed. The idea should be that someone can do `Function('f', commutative=False)`.